skiplang/prelude/src/skstore/Context.sk

```diff
@@ -1569,7 +1572,6 @@ fun runWithGcIntern(
           sync,
           underLockF,
         );
-        gGlobalUnlock();
```

Comment:
Don't we have the same issue in the `CStop` branch?

Reply:
I don't think so, as (in the synchronizer-free case) the lock is now acquired at the beginning, held through all iterations, and released after processing the `CStop()` which triggers the end of the loop.